flink-connector-aws/flink-connector-aws-kinesis-streams/src/main/java/org/apache/flink/connector/kinesis/source/KinesisStreamsSource.java

```diff
@@ -254,31 +252,10 @@ private RetryStrategy createExpBackoffRetryStrategy(
         final BackoffStrategy backoffStrategy =
                 BackoffStrategy.exponentialDelayHalfJitter(initialDelay, maxDelay);
 
-        return StandardRetryStrategy.builder()
+        return SdkDefaultRetryStrategy.standardRetryStrategyBuilder()
```

Comment:
Do we have any thoughts on how this would work with a user defined retryable exceptions improvement like the one in the old connector?

Reply:
> Do we have any thoughts on how this would work with a user defined retryable exceptions improvement like the one in the old connector?

Good qn - we can add more predicates by calling `retryOnException()`, but we won't be able to override the ones already marked as non-retryable in the SDK defaults, because I think it does eager checks for non-retryable errors.